### PR TITLE
Add explicit invite to sig-aws-meetings

### PR DESCRIPTION
**Pull request by tsupertramp**
Add small section named Contributing which invites Users to attend meetings and link
them for more Informations to community-membership.md.

Greetings, Thomas

CC as discussed: @chrislovecnm

### CODE REVIEW

File: sig-aws/README.md [hunk 1]
```diff
@@ -37,5 +37,8 @@ Note that the links to display team membership will only work if you are a membe
 | @kubernetes/sig-aws-misc | [link](https://github.com/orgs/kubernetes/teams/sig-aws-misc) | General Discussion |
 
 <!-- BEGIN CUSTOM CONTENT -->
+## Contributing
+Feel free to join our Meetings and help us growing. We depend on new people becoming members and regular code contributors.
```
Comment (chrislovecnm) [inline sig-aws/README.md]: Help us grow? 

We should have a contributing in this directory as well.

File: sig-aws/README.md [hunk 2]
```diff
@@ -37,5 +37,8 @@ Note that the links to display team membership will only work if you are a membe
 | @kubernetes/sig-aws-misc | [link](https://github.com/orgs/kubernetes/teams/sig-aws-misc) | General Discussion |
 
 <!-- BEGIN CUSTOM CONTENT -->
+## Contributing
```
Comment (chrislovecnm) [inline sig-aws/README.md]: So how about a title about the meetings? What I head from what you said is that you want to encourage people to interact on the calls.

File: sig-aws/README.md (hunk 1 above)
Comment (tsupertramp) [reply sig-aws/README.md]: I tried to mix contribution invitation + meeting invitation.

Will split it into two Parts. A CONTRIBUTING.md and some Meeting Invitation.

File: sig-aws/README.md (hunk 2 above)
Comment (tsupertramp) [reply sig-aws/README.md]: Will change this as well.

**Comment by tsupertramp** [comment]
Not ready yet. Sorry. Just wanted to push my daily work.

**Comment by chrislovecnm** [comment]
Will put hold on this ... you can do “/hold cancel” when you are ready.
/hold

**Comment by tsupertramp** [comment]
Rechecked Links and Texts and improved one small thing - I tried to copy/adapt other texts from this Repository. This text would have answered the questions which i had some weeks ago.

**Comment by chrislovecnm** [comment]
Looks great let’s iterate

/lgtm

**Pull request merged**